samples/CMakeLists.txt

```diff
@@ -19,12 +19,16 @@ set(AMBER_SOURCES
     config_helper.cc
     log.cc
     ppm.cc
-    png.cc
     timestamp.cc
     ${CMAKE_BINARY_DIR}/src/build-versions.h.fake
 )
 
-set(AMBER_EXTRA_LIBS "lodepng")
+set(AMBER_EXTRA_LIBS "")
+
+if (EXISTS ${PROJECT_SOURCE_DIR}/third_party/lodepng)
```

Comment:
Instead of doing this based on directory existence, it should be a build time option similar to shaderc with `AMBER_DISABLE_SHADERC`